src/Makefile

```diff
@@ -8,16 +8,24 @@ PATHLIB = $(PATHDIR)/lib
 XLIBS = -lgfortran
 
 CC    = gcc
+# CCwin32 = "C:\Program\ Files (x86)\mingw-w64\i686-5.3.0-posix-dwarf-rt_v4-rev0\mingw32\bin\gcc.exe"
```

Comment:
Julia uses the win32 thread model and sjlj (32 bit) / seh (64 bit) exceptions, this may be okay for now but in case it causes any issues later it's something to keep in mind.